### PR TITLE
feat: use deployment helper when available

### PR DESCRIPTION
**Pull request by shyim**

### CODE REVIEW

**Review by keulinho — APPROVED** [review]

File: fpm/rootfs/setup [hunk 1]
```diff
@@ -1,8 +1,25 @@
 #!/usr/bin/env sh
 
 set -e
-set -x
 
+if [[ -e /var/www/html/vendor/bin/shopware-deployment-helper ]]; then
+    cd /var/www/html
+    exec /var/www/html/vendor/bin/shopware-deployment-helper run
+fi
```
Comment (mitelg) [inline fpm/rootfs/setup:8]: don't we need to exit after that? :thinking:

Comment (shyim) [reply fpm/rootfs/setup:8]: exec replaces the current process and stops the execution

Comment (mitelg) [reply fpm/rootfs/setup:8]: TIL :grin:

File: fpm/rootfs/setup [hunk 2]
```diff
@@ -1,8 +1,25 @@
 #!/usr/bin/env sh
 
 set -e
-set -x
 
+if [[ -e /var/www/html/vendor/bin/shopware-deployment-helper ]]; then
+    cd /var/www/html
+    exec /var/www/html/vendor/bin/shopware-deployment-helper run
```
Comment (mitelg) [inline fpm/rootfs/setup:7]: if we `cd` in the `/var/www/html` first, do we still need the full path on the exec command?

**Pull request merged**